src/errors/test-run/templates.js

```diff
@@ -373,7 +373,7 @@ export default {
     `,
 
     [TEST_RUN_ERRORS.multipleWindowsModeIsDisabledError]: err => `
-        Multi window mode is disabled. Remove the "--disable-multiple-windows" CLI flag or set the "disableMultipleWindows" option to "false" in the API to use the "${err.methodName}" method.
+        Multi window mode is disabled. Disable the "disableMultipleWindows" option to use the "${err.methodName}" method.
```

Comment:
`Disable the "disableMultipleWindows"...`

It's tautology. 

What about such variant:
```js
Multi window mode is disabled. To use the "${err.methodName}" method you should remove the "disableMultipleWindows" option.
```

Reply:
```suggestion
        Multi-window mode is disabled.  To use the "${err.methodName}" method, remove the "disableMultipleWindows" option.
```